binary128/binary128_test.go

```diff
@@ -56,13 +56,13 @@ func TestNewFromFloat64(t *testing.T) {
 		// Special numbers.
 
 		// +NaN
-		{uint64Float: 0x7FF8000000000000, a: 0x7fff800000000000, b: 0, str: "+Nan not equal"},
-		// -NaN
-		{uint64Float: 0xFFF8000000000000, a: 0xffff800000000000, b: 0, str: "-Nan not equal"},
-		// +Inf
 		{uint64Float: 0x7FF0000000000000, a: 0x7fff000000000000, b: 0, str: "+Inf not equal"},
-		// -Inf
+		// -Nan
```

Comment:
Is this comment correct now? It does not seem to be consistent with `str`: *"-Inf not equal"*